src/main/resources/locale_de_DE.properties

```
@@ -12,6 +12,13 @@ command.channel.sub.set=Setzen des Reputationskanals
 command.channel.sub.set.set=Setzt den folgenden Kanal als Reputationskanal:\n%CHANNEL%
 command.channel.sub.whitelist.false=In gelisteten Kanälen kann keine Reputation gesammelt werden.
 command.channel.sub.whitelist.true=Nur in gelisteten Kanälen kann Reputation gesammelt werden.
+command.dashboard.description=Erhalten Informationen über diesen Server.
```

Other command descriptions don't directly address the reader. Additionally, this is grammatically wrong.
```suggestion
command.dashboard.description=Zeigt Informationen über diesen Server an.
```